src/s2mMuscleHillTypeThelenFatigable.cpp

```diff
@@ -63,6 +63,19 @@ void s2mMuscleHillTypeThelenFatigable::computeFlCE(const s2mMuscleStateActual &E
     m_FlCE *= m_fatigueState->activeFibers();
 }
 
+std::shared_ptr<s2mMuscleFatigueState> s2mMuscleHillTypeThelenFatigable::fatigueState()
```

Comment:
This is already done by the mother s2mMuscleFatigable class. Therefore, it should not appear here